packages/pipeline-editor/src/TabbedPanelLayout/index.tsx

```diff
@@ -24,6 +24,7 @@ interface Props {
   tabs: {
     id: string;
     label: string;
+    tooltip: string;
```

Comment:
can we rename this to `title`, since it's setting the title

Reply:
sure